diff --git a/data/appdata/org.gnome.Calendar.appdata.xml.in b/data/appdata/org.gnome.Calendar.appdata.xml.in
index 2c26340..e61104f 100644
--- a/data/appdata/org.gnome.Calendar.appdata.xml.in
+++ b/data/appdata/org.gnome.Calendar.appdata.xml.in
@@ -33,7 +33,7 @@
     </screenshot>
     <screenshot>
       <caption>Calendar management</caption>
-      <image width="1200" height="675">https://git.gnome.org/browse/gnome-calendar/plain/data/appdata/calendar-management.png</image>
+      <image width="1202" height="681">https://git.gnome.org/browse/gnome-calendar/plain/data/appdata/calendar-management.png</image>
     </screenshot>
   </screenshots>
 
